src/prefect/utilities/notifications/notifications.py

```diff
@@ -125,7 +125,12 @@ def slack_message_formatter(
     tracked_obj: TrackedObjectType,
     state: "prefect.engine.state.State",
     backend_info: bool = True,
+    template: str = None,
+    template_vars: Mapping = None,
 ) -> dict:
+    template = template or f"{tracked_obj} is now in a {state} state"
+    template_vars = template_vars or {}
```

Comment:
We may want to update `template_vars` with `tracked_obj` and `state` here so there cannot be collisions in the `.format(...)` call